pkg/handlers/v1/cloud_fetch.go

```diff
@@ -108,6 +117,46 @@ func (h *CloudFetchByHostnameHandler) Handle(ctx context.Context, input CloudAss
 	return extractOutput(assets), nil
 }
 
+// CloudFetchAllByTimestampHandler defines a lambda handler for bulk fetching cloud assets known at specific point in time
+type CloudFetchAllByTimestampHandler struct {
+	LogFn   domain.LogFn
+	StatFn  domain.StatFn
+	Fetcher domain.CloudAssetAllByTimestampFetcher
+}
+
+// Handle handles fetching cloud assets by hostname
+func (h *CloudFetchAllByTimestampHandler) Handle(ctx context.Context, input CloudAssetFetchAllByTimestampParameters) (CloudAssets, error) {
+	logger := h.LogFn(ctx)
+
+	ts, e := time.Parse(time.RFC3339Nano, input.Timestamp)
+	if e != nil {
+		logger.Info(logs.InvalidInput{Reason: e.Error()})
+		return CloudAssets{}, InvalidInput{Field: "time", Cause: e}
+	}
+
+	if input.Count == nil {
```

Comment:
question: could the `input.Count` be equal to 0? wouldn't this check miss it if the count was 0?

Reply:
```    schema:
            type: "integer"
            minimum: 1
```
the API specs would not allow zero